File: CS/MyLocalizer.cs
using System;
using System.Drawing;
using System.Collections;
using System.ComponentModel;
using System.Windows.Forms;
using System.Data;
using DevExpress.XtraEditors.Controls;

namespace LocalizedText {
    public class MyLocalizer : Localizer {
        public override string GetLocalizedString(StringId id) {
            switch(id) {
                case StringId.NavigatorTextStringFormat: return "Item {0} of {1}";
            }
            return base.GetLocalizedString(id);
        }
    }
}

File: VB/MyLocalizer.vb
Imports Microsoft.VisualBasic
Imports System
Imports System.Drawing
Imports System.Collections
Imports System.ComponentModel
Imports System.Windows.Forms
Imports System.Data
Imports DevExpress.XtraEditors.Controls

Namespace LocalizedText
	Public Class MyLocalizer
		Inherits Localizer
		Public Overrides Function GetLocalizedString(ByVal id As StringId) As String
			Select Case id
				Case StringId.NavigatorTextStringFormat
					Return "Item {0} of {1}"
			End Select
			Return MyBase.GetLocalizedString(id)
		End Function
	End Class
End Namespace
